wordpress-mobile/WordPress-iOS
# Replace `AccountSettingsService.context` with `CoreDataStack`

Use `CoreDataStack` instead of `NSManagedObjectContext` in `AccountSettingsService`. See the "Issue" and "Proposal" sections of https://github.com/wordpress-mobile/WordPress-iOS/pull/19893 for rational behind this change.

## Test instructions

Tap the user avatar icon, go to "Account Settings" page, verify all settings can be changed.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

3. What automated tests I added (or what prevented me from doing so)
A couple of unit tests to verify changes are saved or reverted correctly.

PR submission checklist:
